Samples/Swift/DaysUntilBirthday/Shared/Services/VerificationLoader.swift

```diff
@@ -32,7 +32,7 @@ final class VerificationLoader: ObservableObject {
   /// Verifies the user's age based upon the selected account.
```

Comment:
We don't verify a user's precise age. I think we should update the doc comment and method name to indicate that this method verifies that a user's age is over 18. 

`verifyUserAgeOver18() { ... }`

Reply:
Done.